README.md

```diff
@@ -81,7 +81,7 @@ Once you have valid configuration and credentials you can run tests with [npm][7
   npm test
 ```
 
-#### Author: [Charlie Robbins](http://blog.nodejitsu.com)
+#### Author: [Shweta Jain](https://www.loggly.com/)
```

Comment:
@mchaudhary @mostlyjason I have made changes in the README.md file as per our discussion on  
Author name. Please review..